### PR TITLE
Refactor/mbl misc updates

### PR DESCRIPTION
**Pull request by macintoshpie**
## Background
MBL requires some edits after running modelica-fmt. This attempts to address a few of those changes they require.
This PR does not address the need to break on long lines inside of non-model level annotations, which is known to be desired (see here: https://github.com/lbl-srg/modelica-buildings/commit/7a2cc97092680071e7d18b881ee89fe9a890d32d#diff-cc373b9f69fcbc72e7c1ac989931c40bbd538461512f666dc9ca977bc28018a8R51-R54).

## Changes
- add new term to grammar to indent external function call args (see https://github.com/lbl-srg/modelica-buildings/commit/7a2cc97092680071e7d18b881ee89fe9a890d32d#diff-cc373b9f69fcbc72e7c1ac989931c40bbd538461512f666dc9ca977bc28018a8R37)
- add more model annotations to gmt-building example to show how indentation works (was already implemented in current modelica-fmt, but see here for older results: https://github.com/lbl-srg/modelica-buildings/commit/7a2cc97092680071e7d18b881ee89fe9a890d32d#diff-42232ab126f777675052e10ce578964a38351e3c093f096ea97bdc0cb362076aR107)

### CODE REVIEW

File: examples/gmt-building-out.mo [hunk 1]
```diff
@@ -185,6 +185,35 @@ February 21, 2020, by Antoine Gautier:<br/>
 First implementation.
 </li>
 </ul>
-</html>"));
+</html>"),
+    Icon(
+      graphics={
+        Rectangle(
```
Comment (nllong) [inline examples/gmt-building-out.mo:191]: very nice

File: examples/functions-out.mo [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    output MyClass adapter;
+  external "C" adapter=ExternalFunctionCall(
+    param1,
+    param2,
+    param3);
+  end constructor;
+  function destructor
+    "Some comment"
+    extends Modelica.Icons.Function;
+    input Integer input2;
```
Comment (nllong) [inline examples/functions-out.mo:19]: looks good!

File: main.go [hunk 3]
```diff
@@ -77,6 +77,7 @@ func main() {
 	}
 	if flag.NArg() == 0 {
 		fmt.Fprintln(os.Stderr, "error: must provide at least one file or directory")
+		usage()
```
Comment (nllong) [inline main.go:80]: usage reports the command line arguments?

**Review by nllong — APPROVED** [review]
i'm no a go developer, but the output files look right. Good jorrb.

File: main.go (hunk 3 above)
Comment (macintoshpie) [reply main.go:80]: Yeah, previously it would just say "error: ..." but having usage below the error will be helpful

**Pull request merged**